### DISCUSSION TITLE
Did I do something wrong when installing Homebrew?

### DISCUSSION THREAD

**Discussion by bryanjvela** [Homebrew/discussions#4045] [Getting started] [answered]
Hello,

I'm relatively new to the terminal and when I launched the terminal today I noticed this error here:

<img width="569" alt="Screenshot 2022-12-14 at 12 52 11 AM" src="https://user-images.githubusercontent.com/80439356/207550135-7465c040-a176-421d-8855-0666ab00ce67.png">

I'm not quite sure what this means or how to go about fixing this error.
Any help or advice is extremely appreciated and welcome! Have a nice day!

**Comment by gromgit** [comment]
What's the output of `cat ~/.zprofile`?

**Reply by bryanjvela** [reply]
Thanks for answering, this is what appears when entering that command:


<img width="964" alt="Screenshot 2022-12-14 at 1 55 03 AM" src="https://user-images.githubusercontent.com/80439356/207564024-4f67b2f9-c211-4015-93bd-f997cfe9b179.png">

**Comment by carlocab** [comment, marked as answer]
Do
```
open ~/.zprofile
```
and modify the file so that it reads
```
# Added by Toolbox App
export PATH="$PATH:/Users/bryanvela/Library/Application Support/JetBrains/Toolbox/scripts"
# Set PATH, MANPATH, etc., for Homebrew.
eval "$(/opt/homebrew/bin/brew shellenv)"
```

The other lines are extraneous and can be removed. Save the file and restart your terminal.

Also, please [don't use screenshots](https://meta.stackoverflow.com/a/285557) when sharing the output of commands in your terminal. They're often difficult to read and impossible to reuse (e.g. when looking up error messages). Instead, copy the text here and wrap them in three back-ticks (```) on the line before and on the line after to format them properly.

**Reply by bryanjvela** [reply]
Thanks Carlo, for the help and advice, I appreciate it.

After opening and modifying the file to your recommendation, I'm now getting this message when opening the terminal:
```
/Users/bryanvela/.zprofile:6: permission denied: /opt
bryanvela@Bryans-MBP-3 ~ % 
``` 
I'm not sure what I did exactly that resulted in these errors. Any help or advance is extremely appreciated, thanks!

**Reply by gromgit** [reply]
There's an extra space in the original recommendation. The last line should read:
```
eval "$(/opt/homebrew/bin/brew shellenv)"
```

**Reply by bryanjvela** [reply]
Thanks, Adrian and Carlo for the help, I'm not getting any errors when opening the terminal now. 
I hope you both have a nice rest of your day!
